tests/test_stats.py

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env python
+# encoding: utf-8
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+"""
+This module implements test coverage for the stats functions in stats.py.
+"""
+import itertools
```

Comment:
Can we add more tests, with some different distributions? We could add, for example:

- Normally distributed histograms
- Skewed distributions
- Uniformly distributed histograms

And combinations therein. We can create them randomly, and check that our result is within scipy's by some value.